src/manual/Usage/Services.md

```diff
@@ -0,0 +1,26 @@
+# Running ClamAV Services
+
+Table Of Contents
+
+- [Windows Services](#windows)
+<!--- - [Unix Services](#unix) --->
+
+## Windows Services
+
```

Comment:
We need to note that the Windows Services feature will require ClamAV 0.104 or newer.